src/tmplits/tmplits-tabs/module.js

```diff
@@ -0,0 +1,184 @@
+import * as util from "../tmplits-utilities/module.js"
+
+//Define the style template
+
+
+const styleTemplate = document.createElement('template');
+styleTemplate.innerHTML = 
+`
+<style>
+.tmplit-tab-control{
+    background-color: var(--tmplit-tab-color, var( --md-sys-color-surface-container, ));
+    padding: var(--tmplit-tab-padding, 10px);
+    margin: var(--tmplit-tab-margin, 10px);
+}
+.tmplit-frame{
+    border: var(--tmplit-tab-frame-border, 1px solid black);
+    border-radius: var(--tmplit-tab-frame-radius, 10px);
+    padding: var(--tmplit-tab-frame-padding, 10px);
+    color : var(--tmplit-tab-frame-text-color, var( --md-sys-color-on-secondary-container,  #385E72));
+    background-color: var(--tmplit-tab-frame-color, var( --md-sys-color-secondary-container, #D9E4EC));
```

Comment:
I am not sure about this but I was thinking that since we could have containers inside of tabs quite often maybe this would be better as a surface container? That way the containers on top stand out more? Maybe just need to play with this as we go along.